stylix/palette.nix

```
@@ -74,6 +80,26 @@ in {
         doesnt fix your monitor perfectly
       '';
     };
+    imageEditor = {
+      enable = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = ''
+          Update Wallpaper by applying a lut filter to the image
+        '';
+      };
```

Reuse `lib.mkEnableOption` and generalize `description`:

```suggestion
  imageEditor =
    lib.mkEnabledOption "'stylix.image' being edited"
    // {default = false;};
```